### PR TITLE
3.9.0

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.9.0](https://github.com/sonata-project/SonataNewsBundle/compare/3.8.0...3.9.0) - 2019-09-02
+### Added
+- Add missing `twig/twig` required in `composer.json` with versions `^1.35 || ^2.4`
+- Added events to the comment process
+- Added a new test class SonataNewsExtensionTest. Inspired by the
+  SonataMediaExtensionTest, I extended AbstractExtensionTestCase to test if the
```
Comment (core23) [inline CHANGELOG.md]: > I extended AbstractExtensionTestCase to test [..]

We should reword this and remove the `I extended` here

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,27 @@
...
+association with media, user, tag, collection have been created according to a
+configuration
+- A dependency to [matthiasnoback/symfony-dependency-injection-test
+](https://packagist.org/packages/matthiasnoback/symfony-dependency-injection-test)
+- Added support for flex
```
Comment (core23) [inline CHANGELOG.md]: Not yet. We need a recipe which is not possible for now :(

Comment (greg0ire) [reply CHANGELOG.md]: Do I just remove that line then?

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,26 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.9.0](https://github.com/sonata-project/SonataNewsBundle/compare/3.8.0...3.9.0) - 2019-09-02
+### Added
+- Add missing `twig/twig` required in `composer.json` with versions `^1.35 || ^2.4`
+- Added events to the comment process
+- Added a new test class `SonataNewsExtensionTest`. Inspired by the
```
Comment (core23) [inline CHANGELOG.md]: This is a test class. You could remove this

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,26 @@
...
+  SonataMediaExtensionTest,  `AbstractExtensionTestCase` was extended to test
+if the association with media, user, tag, collection have been created
+according to a configuration.
+- A dependency to [matthiasnoback/symfony-dependency-injection-test
```
Comment (core23) [inline CHANGELOG.md]: This too

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,26 @@
...
+](https://packagist.org/packages/matthiasnoback/symfony-dependency-injection-test)
+
+### Fixed
+- deprecation notice about using namespaced classes from `\Twig\`
+
+### Changed
+- `src/DependencyInjection/SonataNewsExtension.php` registerDoctrineMapping to
```
Comment (core23) [inline CHANGELOG.md]: What?

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,26 @@
...
+  add a `trigger_error` message
+- Improved support for different db drivers
+- The default driver is now `no_driver`
```
Comment (core23) [inline CHANGELOG.md]: The default is still `doctrine_orm`

Comment (greg0ire) [reply CHANGELOG.md]: So do I remove this line too?

File: CHANGELOG.md (hunk 5 above)
Comment (greg0ire) [reply CHANGELOG.md]: Comes from this PR: https://github.com/sonata-project/SonataNewsBundle/pull/518

Comment (core23) [reply CHANGELOG.md]: I think you could just use the PR title:
```diff
-- `src/DependencyInjection/SonataNewsExtension.php` registerDoctrineMapping to
-   add a `trigger_error` message
+ Trigger an error if a class declaration is missing in the configuration
```

**Pull request merged**